doc/examples/applications/plot_coins_segmentation.py

```diff
@@ -54,9 +53,9 @@
 # segmentation. To do this, we first get the edges of features using the
 # Canny edge-detector.
 
-from skimage.feature import canny
+import skimage as ski
 
```

Comment:
```suggestion
```
You have already imported `skimage` (line 14), so you don't need to import it again.